etl/load.py

```diff
@@ -51,14 +52,52 @@ def _generate_upsert_sql(mon_loc):
         f"MERGE INTO GW_DATA_PORTAL.WELL_REGISTRY_STG a "
         f"USING (SELECT '{mon_loc['AGENCY_CD']}' AGENCY_CD, '{mon_loc['SITE_NO']}' "
         f"SITE_NO FROM DUAL) b ON (a.AGENCY_CD = b.AGENCY_CD AND a.SITE_NO = b.SITE_NO) "
-        f"WHEN MATCHED THEN UPDATE SET {update_query} WHEN NOT MATCHED THEN INSERT ({all_columns}) VALUES ({all_values})"
+        f"WHEN MATCHED THEN UPDATE SET {update_query} "
+        f"WHEN NOT MATCHED THEN INSERT ({all_columns}) VALUES ({all_values})"
```

Comment:
How big can {all_values} get? It probably is ok. Do not know about Oracle. For Postgres, if the size is greater than
about 2 Gigs,  the maximum length of a query that PostgreSQL supports would be exceeded.

Reply:
this is just one row in the registry table. a few KB max.